src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx

```
@@ -20,7 +20,27 @@
       }
     }
   }, [imagePreview])
-
+  useEffect(() => {
+    const savedImage = localStorage.getItem(STORAGE_KEY)
+    if (savedImage) {
+      setImagePreview(savedImage)
+    }
+    return () => {
+      if (imagePreview && imagePreview.startsWith('blob:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)